src/presentation/pages/Home.vue

```diff
@@ -1,8 +1,25 @@
 <template>
-  <div :class="$style.page">
+  <div>
     <h1>{{ $t('home.title') }}</h1>
     <div v-if="user">
-      <NoteList />
+      <div v-if="noteList">
+        <div :class="$style.noteList">
+          <div
+            v-for="note in noteList.items"
+            :key="note.id"
+          >
+            <CardVertical
+              :title="getTitle(note.content)"
+              :updated-at="getUpdateTime(note.updatedAt!)"
+              @click="router.push(`/note/${note.id}`)"
```

Comment:
use `RouterLink` component instead of js-click handler

https://router.vuejs.org/guide/